src/components/chat-message.tsx

```diff
@@ -26,26 +20,58 @@ export interface ChatMessageProps {
 		time: string
 	) => void;
 	isUnauthenticated?: boolean;
-	failedMessage?: boolean;
 }
 
+export const MarkDownWithCode = ( {
+	message,
+	updateMessage,
+	siteId,
+	content,
```

Comment:
Nitpick: if we rename this to `children` we could pass the content from `ChatMessage` as children when rendering it. I.e.:
```
<MarkDownWithCode message={ message } updateMessage={ updateMessage } siteId={ siteId }>
	{ children }
</MarkDownWithCode>
```

Reply:
I didn't do it like that, as with that specific component, the children are never React.Node, but it's always a string.